setup.py

```diff
@@ -15,4 +15,4 @@
 import setuptools
 
 
-setuptools.setup(setup_requires=["pbr>=2.0.0"], pbr=True)
+setuptools.setup(setup_requires=["pbr>=2.0.0"], version="v1.0.0", pbr=True)
```

Comment:
I think we can remove all of the pbr-related things now that it's not actually being used in the release process anymore.

We should also update and/or remove the `release.sh` script.

Reply:
I removed the release.sh. Lmk if there's any pbr references I ought to remove